src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -73,12 +73,75 @@ def ssl_client_credentials(root_certificates, private_key, certificate_chain):
       certificate chain should be used.
 
   Returns:
-    A ClientCredentials for use with an SSL-enabled Channel.
+    A ChannelCredentials for use with an SSL-enabled Channel.
   """
-  intermediary_low_credentials = _intermediary_low.ClientCredentials(
-      root_certificates, private_key, certificate_chain)
-  return ClientCredentials(
-      intermediary_low_credentials._internal, intermediary_low_credentials)  # pylint: disable=protected-access
+  return ChannelCredentials(_low.channel_credentials_ssl(
+      root_certificates, private_key, certificate_chain))
+
+
+class CallCredentials(object):
+  """A value encapsulating data asserting an identity over an *established*
+  channel. May be composed with ChannelCredentials to always assert identity for
+  every call over that channel.
+
+  This class and its instances have no supported interface - it exists to define
+  the type of its instances and its instances exist to be passed to other
+  functions.
+  """
+
+  def __init__(self, low_credentials):
+    self._low_credentials = low_credentials
+
+
+def metadata_call_credentials(metadata_callback, name=None):
+  """Construct call credentials from an auth metadata plugin.
+
+  Args:
+    metadata_callback (callable): Callback accepting an
```

Comment:
(1) Identify the type of a parameter in the prose on the right of the colon, not with the parameter name on the left side of the colon. (https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments contains the canonical sample to follow.)

(2) "callable" is no more a complete type description than "set", "tuple", or "dict". Just as a description of a dictionary should read "a dictionary from integers to bytetrings", the type description here should probably be something like "a callable taking an X and a Y and either returning a Z or raising a Q".